modules/nf-core/gprofiler2/gost/meta.yml

```diff
@@ -0,0 +1,110 @@
+name: "gprofiler2_gost"
+description: runs a functional enrichment analysis with gprofiler2
+keywords:
+  - gene set analysis
+  - enrichment
+  - gprofiler2
+  - gost
+  - gene set
+tools:
+  - "gprofiler2":
+      description: "An R interface corresponding to the 2019 update of g:Profiler web tool."
+      homepage: "https://biit.cs.ut.ee/gprofiler/page/r"
+      documentation: "https://rdrr.io/cran/gprofiler2/"
+      tool_dev_url: "https://gl.cs.ut.ee/biit/r-gprofiler2"
+      doi: "10.1093/nar/gkad347"
+      licence: ["GPL v2"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing contrast information, e.g. [ variable:'treatment', reference:'treated', control:'saline', blocking:'' ]
+  - de_file:
+      type: file
+      pattern: "*.{csv,tsv}"
+      description: |
+        CSV or TSV-format tabular file with differential analysis outputs
+  - contrast_variable:
+      type: string
+      description: |
+        The contrast variable that is being investigated in DE analysis, e.g. "treatment".
+  - reference:
+      type: string
+      description: |
+        The contrast level of the reference samples, e.g. "control"
+  - target:
+      type: string
+      description: |
+        The contrast level of the target samples, e.g. "treated"
+  - organism:
+      type: string
+      description: |
+        The name of the analyzed organism consisting of the first letter of the genus and the entire species name, e.g. for homo sapiens: hsapiens
+  - sources:
+      type: string
+      description: |
+        Comma-separated list of data sources to search for pathways, e.g. 'GO,REAC'. Check https://rdrr.io/cran/gprofiler2/man/gost.html for a list of available sources
+  - background_file:
+      type: file
+      pattern: "*.{csv,tsv,txt}"
+      description: |
+        Path to a CSV/TSV/TXT file listing gene IDs that should be used as the background (will override count_file). This can be an expression matrix (see also background_column parameter); if so, will only consider those genes with an expression value > 0 in at least one sample. Alternatively, this can be a TXT file containing only a list of gene IDs.
+  - gmt_file:
+      type: file
+      pattern: "*.{gmt}"
```

Comment:
Don't need curly braces in the pattern when only one option

Reply:
Oops, you are right!